digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/dao/RoadwayChangesDAOSpec.scala

```diff
@@ -95,6 +95,71 @@ class RoadwayChangesDAOSpec extends FunSuite with Matchers {
     }
   }
 
+  test("Test RoadwayChangesDAO().insertDeltaToRoadChangeTable() When a road is transfered to an other road with reverse changes should have one road reversed and the others roadnumber changed .") {
```

Comment:
En ymmärrä testin kuvausta. En näe esim. onko ajatuskatko ennen vai jälkeen "changes" -sanaa.
Ja sitten siis, millä muilla ("the others") pitäis olla roadnumber changed?

JOS tulkitsin katkon oikeaan kohtaan, niin p.o. esim.
"When a [part of] road is transferred to be a part of another road so that the transferred [part of] the road ends up to be reversed, Should have one [part of the] road reversed, and the others should have roadnumber changed ."

Reply:
(Yleisesti: Täällä on testi! \o/ <3 )